content/2_Bronze/Time_Comp.mdx

```diff
@@ -451,3 +452,74 @@ triplets.
 <!-- Bitsets are significantly faster (and so are bitwise operations vs iterating through bits). I don't want to make it sound insignificant -->
 
 For now, don't worry about optimizing constant factors -- just be aware of them.
+
+## Quiz
+<Quiz>
+  <Quiz.Question>
+    What's time complexity?
+    <Quiz.Answer>
+      Measuring the amount of memory an algorithm consumes.
+      <Quiz.Explanation>
+        This is known as space complexity.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Measuring the number of operations an algorithm performs.
+      <Quiz.Explanation>
+        You'd always want your algorithm to run as fast as possible!
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      The time it takes you to solve a problem.
+      <Quiz.Explanation>
+        
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    What time complexity would a mathematical formula take?
+    <Quiz.Answer correct>
+      $\mathcal{O}(1)$
+      <Quiz.Explanation>
+        No matter how big a certain number is, a mathematical formula would execute the same number of operations.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      $\mathcal{O}(N)$
+      <Quiz.Explanation>
+        Would this algorithm take more steps because of a larger number?

```

Comment:
technically for something like addition it does (for example, big int addition is slower) but i don't think it matters that much

Reply:
ig big int addition is slower, but it doesn't take require any more calculations, right?
should i change steps to like calculations or smth?

Reply:
ignore this, it doesn't really matter